workable-module-connector/0.1/antora.yml

```diff
@@ -0,0 +1,18 @@
+name: workable-module-connector
+version: '0.1'
+display_version: 0.1.0-BETA (Mule 4)
+title: Workable Module Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables you to connect to the Workable Module API.
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule4-workable-module-connector
```

Comment:
Update the antora to include the runtime version, like how I did here:
https://github.com/mulesoft/docs-connectors/pull/3058/commits/aba5c34f5455252b1a65838223c961425039aaac